pylint_odoo/test/main.py

```diff
@@ -9,6 +9,7 @@
 
 
 EXPECTED_ERRORS = 52
+EXPECTED_ERRORS += 2  # Errors due to C8106: Wrong version format
```

Comment:
@max3903 
@pedrobaeza indicated me to do so because that way multiple PRs don't get affected each other and the number seems less random.